PPAK_CVS/Sources/AppDelegate.swift

```diff
@@ -23,11 +32,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // Set Firebase
     FirebaseApp.configure()
 
+    // FCM
+    Messaging.messaging().delegate = self
+    Messaging.messaging().token { token, error in
+      if let error = error {
+        print("Error fetching FCM registration token: \(error)")
+      } else if let token = token {
+        print("FCM registration token: \(token)")
+      }
+    }
+
+    // User Notification Authorization
+    let authOptions: UNAuthorizationOptions = [.alert, .badge, .sound]
+    UNUserNotificationCenter.current().requestAuthorization(
+      options: authOptions,
+      completionHandler: { _, error in
+        print("Error request notifications Authorization: \(error.debugDescription)")
+      }
+    )
```

Comment:
클로저는 밖으로 빼주시면 좋겠습니다!
```suggestion
    UNUserNotificationCenter.current().requestAuthorization(options: authOptions) { _, error in
        print("Error request notifications Authorization: \(error.debugDescription)")
      }
```

Reply:
헉 그렇네요! 알려주셔서 감사합니다 :)